src/bundle/ui-dev/src/modules/universal-discovery/content.create.tab.module.js

```diff
@@ -36,8 +30,16 @@ const ContentCreateTabModule = () => {
     const [selectedLocations, dispatchSelectedLocationsAction] = useContext(SelectedLocationsContext);
     const [loadedLocationsMap, dispatchLoadedLocationsAction] = useContext(LoadedLocationsMapContext);
     const [multiple] = useContext(MultipleConfigContext);
-    const iframeUrl = generateIframeUrl(contentOnTheFlyData);
     const iframeRef = createRef();
+    const iframeUrl = () => {
```

Comment:
maybe `getIframeUrl`/generate ? as just `iframeUrl` sugests normal variable, not func